### ISSUE TITLE
TODOs in JohnTravoltageView

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/john-travoltage#138]
```js
      //TODO: Once https://github.com/phetsims/john-travoltage/issues/98 has been addressed, update how the peerIDs
      //are added/referenced by the view.
      peerIDs: {
        alert: 'assertive-alert',
        status: 'polite-status'
      }
```
and
```js
    // unhide the aria-live elements for use in this sim
    // TODO: should be handled in common code
    document.getElementById( 'aria-live-elements' ).hidden = false;
```

Code review for https://github.com/phetsims/john-travoltage/issues/128

**Comment by jessegreenberg** [comment]
These were handled in #171 (and its related issues). The TODOs are no longer there.
